google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -123,7 +124,7 @@
    @Fallback(NullOnNotFoundOr404.class)
    @Nullable
    GoogleCloudStorageObject getObject(@PathParam("bucket") String bucketName, @PathParam("object") String objectName,
-            GetObjectOptions options);
+         HttpRequestOptions options);
```

Comment:
After this commit, we have no other uses of `GetObjectOptions`.  Are these generation and projection options not needed?  If so should we remove that class entirely?